src/components/Calculator.js

```diff
@@ -25,22 +44,42 @@ function Calculator() {
   return (
     <div className="Application">
       <div className="calc">
-        <Display value={0} />
+        <Display objState={objState} />
         <div className="operands-container">
           <div className="left-operand">
-            <button type="button">AC</button>
-            <button type="button">+/-</button>
-            <button type="button">%</button>
+            <button type="button" onClick={() => handleClick('AC')}>
+              AC
+            </button>
+            <button type="button" onClick={() => handleClick('+/-')}>
+              +/-
+            </button>
+            <button type="button" onClick={() => handleClick('%')}>
+              %
+            </button>
             {createDigits()}
-            <button type="button">0</button>
-            <button type="button">.</button>
+            <button type="button" onClick={() => handleClick('0')}>
+              0
+            </button>
+            <button type="button" onClick={() => handleClick('.')}>
+              .
+            </button>
           </div>
           <div className="right-operand">
-            <button type="button">+</button>
-            <button type="button">*</button>
-            <button type="button">-</button>
-            <button type="button">/</button>
-            <button type="button">=</button>
+            <button type="button" onClick={() => handleClick('+')}>
+              +
+            </button>
+            <button type="button" onClick={() => handleClick('*')}>
+              *
+            </button>
+            <button type="button" onClick={() => handleClick('-')}>
+              -
+            </button>
+            <button type="button" onClick={() => handleClick('/')}>
```

Comment:
- [ ] also similar error occurs when I click this button, kindly pass the correct operator sign